src/Umbraco.Core/Migrations/Upgrade/V_8_0_0/DataTypeMigration.cs

```diff
@@ -60,43 +61,42 @@ public override void Migrate()
                     .From<DataTypeDto>()
                     .Where<DataTypeDto>(x => x.NodeId == group.Key)).First();
 
-                var aliases = group.Select(x => x.Alias).Distinct().ToArray();
-                if (aliases.Length == 1 && string.IsNullOrWhiteSpace(aliases[0]))
+                // migrate the preValues to configuration
+                var migrator = _preValueMigrators.GetMigrator(dataType.EditorAlias) ?? new DefaultPreValueMigrator();
+                var config = migrator.GetConfiguration(dataType.NodeId, dataType.EditorAlias, group.ToDictionary(x => x.Alias, x => x));
+                var json = JsonConvert.SerializeObject(config);
+
+                // validate - and kill the migration if it fails
+                var newAlias = migrator.GetNewAlias(dataType.EditorAlias);
+                if (newAlias == null)
+                {
+                    _logger.Warn<DataTypeMigration>($"Skipping validation of configuration for data type {dataType.NodeId} : {dataType.EditorAlias}."
```

Comment:
Any reason to not use structured logging?

Reply:
no, fixing